sources/packages/web/src/services/EducationProgramService.ts

```diff
@@ -13,6 +13,10 @@ export class EducationProgramService {
     return ApiClient.EducationProgram.getProgram(programId);
   }
 
+  public async getInstitutionForProgram(): Promise<any> {
```

Comment:
we can avoid any and use an interface

Reply:
@ann-aot  for this case, we used any for a purpose, as we do not know if we need more variables from the institution object related to program.